src/person/person.spec.ts

```diff
@@ -29,62 +33,60 @@ const personExamples: IPerson[] = [
     lastName: 'Ron',
     dischargeDay: new Date(2022, 11),
     mail: 'avi.ron@gmail.com',
-    hierarchy: ['Airport', 'Pilots guild', 'captain'],
     job: 'Pilot 1',
     serviceType: SERVICE_TYPE[0],
-    directGroup: dbIdExample[3],
   },
   <IPerson>{
     identityCard: '234567891',
     personalNumber: '3456712',
     firstName: 'Mazal',
     lastName: 'Tov',
     dischargeDay: new Date(2022, 11),
-    hierarchy: ['birthday', 'anniversary'],
     job: 'parent',
-    serviceType: SERVICE_TYPE[0],
-    directGroup: dbIdExample[3],    
+    serviceType: SERVICE_TYPE[0],  
   },
   <IPerson>{
     identityCard: '345678912',
     personalNumber: '4567123',
     firstName: 'Eli',
     lastName: 'Kopter',
     dischargeDay: new Date(2022, 11),
-    hierarchy: ['Airport', 'Pilots guild'],
     job: 'Pilot 2',
     responsibility: RESPONSIBILITY[1],
     responsibilityLocation: dbIdExample[1],
     clearance: '3',
     rank: RANK[0],
     serviceType: SERVICE_TYPE[0],
-    directGroup: dbIdExample[3],
   },
   <IPerson>{
     identityCard: '456789123',
     personalNumber: '5671234',
     firstName: 'Tiki',
     lastName: 'Poor',
     dischargeDay: new Date(2022, 11),
-    hierarchy: ['fashion designer', 'cosmetician guild'],
     job: 'cosmetician 1',
     serviceType: SERVICE_TYPE[0],
-    directGroup: dbIdExample[1],
   },
   <IPerson>{
     identityCard: '567891234',
     personalNumber: '1234567',
     firstName: 'Yonatan',
     lastName: 'Tal',
     dischargeDay: new Date(2022, 11),
-    hierarchy: ['www', 'microsoft', 'github'],
     job: 'Programmer',
     serviceType: SERVICE_TYPE[0],
-    directGroup: dbIdExample[3],
   },
 ];
 
 describe('Persons', () => {
+  // create OG to link with each person.
+  beforeEach(async () => {
```

Comment:
Is there a way to not repeat the code twice. For example put the function in another file and use it in both places that need it